randovania/games/prime1/gui/dialog/prime_cosmetic_patches_dialog.py

```diff
@@ -37,6 +84,51 @@ def persist(value: int):
 
         return persist
 
+    def _persist_suit_color_rotations(self):
+        suit_color_rotations_tuple=(
+            self.power_suit_rotation_field.value(),
+            self.varia_suit_rotation_field.value(),
+            self.gravity_suit_rotation_field.value(),
+            self.phazon_suit_rotation_field.value()
+        )
+        self._cosmetic_patches = dataclasses.replace(self._cosmetic_patches, suit_color_rotations=suit_color_rotations_tuple)
+        self._update_color_squares()
+
+    def _open_color_picker(self):
+        init_color = self._cosmetic_patches.hud_color
+        color = QColorDialog.getColor(QColor(*init_color))
+        
+        if color.isValid():
+            color_tuple = (color.red(), color.green(), color.blue())
+            estimated_ingame_alpha = 255 - ((color_tuple[0] + color_tuple[1] + color_tuple[2])/3)
+            if estimated_ingame_alpha > 150:
+                QMessageBox.warning(self, "Dangerous preset", 
+                    "Be careful, dark colors like this one tend to produce an " \
```

Comment:
surround the string in `()`s, so you don't need the `\` at the end of line.

Reply:
I've been searching on google for 5 minutes to find a better way to write this as I found it extremely ugly lol.
Good to know!